public/react-services/wz-request.tsx

```diff
@@ -42,8 +45,8 @@ export class WzRequest {
         timeout: customTimeout || timeout
       };
       const data = await axios(options);
-      if (data.error) {
-        throw new Error(data.error);
+      if (data['error']) {
+        throw new Error(data['error']);
```

Comment:
Why use `data['error']` instead `data.error`? Is it the same thing, doesn't?

Reply:
this change is because de Axios response type doesn't have the property ** error **, so when I turned the file into .tsx this error appeard. I used ['error'] instead of deleting the code section directly just in case.